pkg/cmd/update/edge_storage/bucket.go

```diff
@@ -17,64 +17,56 @@ import (
 	"github.com/aziontech/azion-cli/utils"
 )
 
-type Fields struct {
-	Name       string
-	EdgeAccess string
-	FileJSON   string
-}
-
 func NewBucket(f *cmdutil.Factory) *cobra.Command {
-	fields := &Fields{}
-
+	bucket := &bucket{
+		factory: f,
+	}
 	cmd := &cobra.Command{
 		Use:           msg.USAGE_BUCKET,
 		Short:         msg.SHORT_DESCRIPTION_CREATE_BUCKET,
 		Long:          msg.LONG_DESCRIPTION_CREATE_BUCKET,
 		SilenceUsage:  true,
 		SilenceErrors: true,
-		Example:       heredoc.Doc(msg.EXAMPLE_CREATE_BUCKET),
-		RunE:          runE(f, fields),
+		Example:       heredoc.Doc(msg.EXAMPLE_UPDATE_BUCKET),
```

Comment:
Go knowledge problems here... 😅 

I don't understand: the funtion is called `NewBucket`, but we're passing an `EXAMPLE_UPDATE_BUCKET` in the example? 🤔

Reply:
great question! The newbucket function is to start all the struct needed for the bucket command and this bucket is the child of the update child, In golang it is common to have a new function to prepare dependencies. @PabloDiehl